cypress/support/step_definitions/validations.js

```diff
@@ -390,3 +390,213 @@ Given(
     });
   }
 );
+
+/**
```

Comment:
https://ccp.sys.comcast.net/browse/FIRECERT-2044 will these changes work with this glue code

Reply:
added the logic to take default values for path and content